challenge_1.exs

```diff
@@ -0,0 +1,41 @@
+defmodule Challenge do
+
+  def num_divisible_by_3_sequence(a, b) when a >= 1 and a <= b and b < 231 do
+      #Initialize the sequence in ["1"] cause the sequence always starts like that
+      sequence = obtain_sequence(["1"], 1, a, b)  #The second parameter is the variable to concat since 1
+      IO.inspect(sequence)
+      divisible_by_3_sequence = Enum.filter(sequence, fn element -> rem(String.to_integer(element), 3) == 0 end)
+      IO.puts("Number of divisibles by 3 elements in the sequence: #{length(divisible_by_3_sequence)}")
+  end
+
+  #Receive invalid parametres with a guard
+  def num_divisible_by_3_sequence(a, b) when a >= 1 or a <= b or b < 231 do
```

Comment:
You can also delete this guard as it is pretty clear this would be the fallback case

Reply:
![code](https://user-images.githubusercontent.com/87148694/197366932-e331adb1-b1fd-42d9-9b95-e9f587e5128d.png)